plugins/optimization-detective/class-od-html-tag-processor.php

```diff
@@ -160,7 +179,31 @@ final class OD_HTML_Tag_Processor {
 	 * @param string $html HTML to process.
 	 */
 	public function __construct( string $html ) {
-		$this->processor = new WP_HTML_Tag_Processor( $html );
+
+		$processor = new class( $html ) extends WP_HTML_Tag_Processor {
+
+			/**
+			 * Appends HTML to the provided bookmark.
+			 *
+			 * @param string $bookmark Bookmark.
+			 * @param string $html     HTML to inject.
+			 * @return bool Whether the HTML was appended.
+			 */
+			public function append_html( string $bookmark, string $html ): bool {
+				if ( ! $this->has_bookmark( $bookmark ) ) {
+					return false;
+				}
+
+				$this->lexical_updates[] = new WP_HTML_Text_Replacement(
+					$this->bookmarks[ $bookmark ]->start,
+					0,
+					$html
+				);
+				return true;
+			}
+		};
+
+		$this->processor = new $processor( $html );
```

Comment:
when creating the anonymous class, isn't it already instantiated? I don't think we want this second `new`

Reply:
Oh, right. Fixed in 7a6debcb9ee097dcf4e9f536b111fcf4644d326a